doc/update-with-bad-bootloader-v2.md

```diff
@@ -0,0 +1,114 @@
+# SMC update with bad bootloader v2
+Author: Eirik Stople
+
+NB: Please read and understand the guide before attempting to use it to recover the SMC.
+
+
+## Simplified explaination and guide
+
+### Affected boards
+
+If you have not updated your SMC yet, and your board is between PR00100 and PR00900, chances are high that you have a "bad bootloader" installed. If this is the case, you will brick your SMC (make it stop working) if you try to update it using the X16. If you do brick your SMC, you have to pop it from the machine and use an external programming tool / Arduino to recover it. See these links for a guide: (or ask for help on Discord)
+- https://discord.com/channels/547559626024157184/1224366756320247942/1224779003601096826
+- https://discord.com/channels/547559626024157184/1224465757786865694
+- https://github.com/X16Community/x16-docs/pull/199
+- https://github.com/X16Community/x16-docs/blob/83455c5664317dbc9f6a7b5662b6a1b997c14c60/X16%20Reference%20-%20Appendix%20G%20-%20SMC%20Update%20and%20Recovery.md
+
+
+### Before you update
+
+If you follow this guide, you will most likely be able to update the SMC without bricking it and having to use an external tool. But you do have to use a small patch wire.
+
+WARNING: Although this guide explains how to perform this update, there is still a risk of something failing, causing a bricked SMC (requiring an external programmer) or even a shorted X16, in case wrong pins are connected. Proceed at your own risk.
+
+This procedure have only been tested with R47 ROM and VERA. Enter "HELP" and ensure that you have these versions:
+- COMMANDER X16 ROM RELEASE R47
+- VERA: V47.0.2
+
+If you have an older version (e.g. R45), start upgrading to R47. See here for a guide: 
+- https://www.facebook.com/groups/CommanderX16/posts/1585423168875438/ or 
+- https://x16community.github.io/faq/articles/r47%20update%20guide.html
+
+IMPORTANT: DO NOT DOWNLOAD AND UPDATE SMC FIRMWARE AT THIS STEP!
+
+Once VERA and ROM have been updated to R47, type "HELP" and note the SMC version. You can also poll the bootloader version using "PRINT I2CPEEK($42,$8E)". If it shows "2", you may either have the good or the bad v2 bootloader. You may want to take a picture of the versions you have before you proceed. The bad bootloader seem to at least be present in PR boards between PR00100 and PR00831, with SMC version 45.1.0. From around PR00900, X16s are delivered with 47.0.0, likely with a good bootloader.
```

Comment:
Do we need to amend this to say R48 is okay? R48 should not have any changes which affect this procedure.

Reply:
The important thing is that the ROM and VERA versions suggested here, should support SMC 47.2.3. I did not test with the original ROM/VERA delivered with my machine, I have mainly tested with ROM/VERA R47. I guess there are no breaking changes in R48, so, we could mention that as well, to keep this document up to date (even though I have not tested this with R48 myself).